airflow/hooks/webhdfs_hook.py

```diff
@@ -55,14 +55,22 @@ def get_conn(self):
         for nn in nn_connections:
             try:
                 self.log.debug('Trying namenode %s', nn.host)
-                connection_str = 'http://{nn.host}:{nn.port}'.format(nn=nn)
+                connection_str = self.connection_url(
+                    'http://',
```

Comment:
This doesn't look like a conn_id, which is what this parameter is being passed as

Reply:
Finally, we get a connection like we should, for example:
url = 'http://host:port;http://host:port'

Reply:
I changed the format